### PR TITLE
Index prefix in config

### PR DESCRIPTION
**Pull request by sgmarghade**
Issue number #96 

1. Modified yml file to read table prefix name.
2. Added field in ElasticSeachConfig.java (Default value as "foxtrot" for backward compatibility )
3. Modified InitializerCommand for ES template creation to read table prefix from config. 
4. Modified scripts/local_es_setup.sh  to read table prefix name from command line argument.

### CODE REVIEW

**Comment by santanusinha** [comment]
@sgmarghade please merge again with 0.3

File: foxtrot-core/src/main/java/com/flipkart/foxtrot/core/querystore/impl/ElasticsearchConfig.java [hunk 1]
```diff
@@ -35,6 +35,7 @@
     @NotNull
     @JsonProperty
     private String cluster;
+    private String tableNamePrefix = "foxtrot";
```
Comment (santanusinha) [inline foxtrot-core/src/main/java/com/flipkart/foxtrot/core/querystore/impl/ElasticsearchConfig.java]: Maybe call this indexPrefix?

File: foxtrot-core/src/main/java/com/flipkart/foxtrot/core/querystore/impl/ElasticsearchUtils.java [hunk 2]
```diff
@@ -46,7 +46,7 @@
     private static final Logger logger = LoggerFactory.getLogger(ElasticsearchUtils.class.getSimpleName());
 
     public static final String TYPE_NAME = "document";
-    public static final String TABLENAME_PREFIX = "foxtrot";
+    public static  String TABLENAME_PREFIX = "foxtrot";
```
Comment (santanusinha) [inline foxtrot-core/src/main/java/com/flipkart/foxtrot/core/querystore/impl/ElasticsearchUtils.java]: DEFAULT_INDEX_PREFIX maybe?

File: foxtrot-core/src/main/java/com/flipkart/foxtrot/core/querystore/impl/ElasticsearchConfig.java (hunk 1 above)
Comment (sgmarghade) [inline foxtrot-core/src/main/java/com/flipkart/foxtrot/core/querystore/impl/ElasticsearchConfig.java]: existing code which creates index prefix looks like this 

return String.format("%s-%s-%s-", ElasticsearchUtils.TABLENAME_PREFIX, table, ElasticsearchUtils.TABLENAME_POSTFIX);

Followed similar convention and named it as table name prefix.

Comment (santanusinha) [inline foxtrot-core/src/main/java/com/flipkart/foxtrot/core/querystore/impl/ElasticsearchConfig.java]: Ok

----
Upset the established order, and everything becomes chaos. I'm an agent of
chaos. Oh and you know the thing about chaos, it's fair.

**Pull request merged**